cli/resources/compute/cluster-system-identity.yml

```diff
@@ -0,0 +1,9 @@
+$schema: https://azuremlschemas.azureedge.net/latest/amlCompute.schema.json 
+name: basic-example
+type: amlcompute
+size: STANDARD_DS3_v2
+min_instances: 0
+max_instances: 2
+idle_time_before_scale_down: 120
+identity:
+  type: system_assigned
```

Comment:
Can we have type as managed ? And it defaults to system assigned. If identity identifier like client id is provided it will be user assigned.

Reply:
This example is based on current compute schema, which expects type: user_assigned or type: system_assigned